themes-default/slim/views/displayShow.mako

```diff
@@ -1,300 +1,187 @@
+                                                    epSize += epResult['file_size']
+                                                    epList.append(epLoc)
+                                        -->
+                                            <span :title="episode.location !== '' ? episode.location : ''" :class="{addQTip: episode.location !== ''}">{{episode.episode}}</span>
+                                        </td>
+                                        <td align="center" class="triggerhighlight">{{episode.scene.absoluteNumber || 0}}</td>
+                                        <td align="center" class="triggerhighlight">
+                                            <input type="text" :placeholder="getSceneNumbering(episode).season + 'x' + getSceneNumbering(episode).episode" size="6" maxlength="8"
+                                                class="sceneSeasonXEpisode form-control input-scene" :data-for-season="episode.season" :data-for-episode="episode.episode"
+                                                :id="'sceneSeasonXEpisode_' + show.id[show.indexer] + '_' + episode.season + '_' + episode.episode"
+                                                title="Change this value if scene numbering differs from the indexer episode numbering. Generally used for non-anime shows."
+                                                :value="getSceneNumbering(episode).season + 'x' + getSceneNumbering(episode).episode"
+                                                style="padding: 0; text-align: center; max-width: 60px;"/>
+                                        </td>
+                                        <td align="center" class="triggerhighlight">
+                                            <input type="text" :placeholder="getSceneAbsoluteNumbering(episode)" size="6" maxlength="8"
+                                                class="sceneAbsolute form-control input-scene" :data-for-absolute="episode.absoluteNumber || 0"
+                                                :id="'sceneSeasonXEpisode_' + show.id[show.indexer] + episode.absoluteNumber"
+                                                title="Change this value if scene absolute numbering differs from the indexer absolute numbering. Generally used for anime shows."
+                                                :value="getSceneAbsoluteNumbering(episode) ? getSceneAbsoluteNumbering(episode) : ''"
+                                                style="padding: 0; text-align: center; max-width: 60px;"/>
+                                        </td>
+                                        <td class="col-name hidden-xs triggerhighlight">
+                                            <!-- < has_plot = 'has-plot' if epResult['description'] else '' > -->
+                                            <plot-info :has-plot="episode.description ? true : false" :show-slug="indexer + id" :season="String(episode.season)" episode="String(episode.episode)"></plot-info>
```

Comment:
`:has-plot="Boolean(episode.description)"` is more readable IMO

Reply:
Agree